### ISSUE TITLE
Custom adapter for Material spinner

### ISSUE THREAD

**Issue by divya-navsoft** [tiper/MaterialSpinner#31]
Hi @tiper really awesome library you've created. I want to add country image flag and country code as fetched from api and add custom adapter in the material spinner and show it when item selected. I have successfully created the custom adapter showing bottom dialog with image flag and code but when I'm selecting one item its only text that showing in Materialspinner. please help.

**Comment by LunevNF** [comment]
ohhh, from 8 jul 2020... facepalm

**Comment by AlfredAbdo** [comment]
> Hi @tiper really awesome library you've created. I want to add country image flag and country code as fetched from api and add custom adapter in the material spinner and show it when item selected. I have successfully created the custom adapter showing bottom dialog with image flag and code but when I'm selecting one item its only text that showing in Materialspinner. please help.

I was trying to find a solution to this, but I don't know how you would properly inflate a view inside a TextInputLayout (or TextInputEditText); remember, the MaterialSpinner view extends TextInputLayout.

A possible solution would be to revert to using a Spinner, but for me, that would defeat the purpose of using this library (since I usually rely on the concept of Live Validation with TextInputLayouts to show required fields in a form-like fragment).

**Comment by sebasira** [comment]
Any updates on this? I'm with the same problem

**Comment by LunevNF** [comment]
> 
> 
> Any updates on this? I'm with the same problem

No, you need to change this lib for result. So, in my project (beta-version) I place ImageView before spinner in horizontal linear layout and when I select some spinner item, I show target image. And in other place of app I use AppCompatAutoCompleteTextView with custom adapter, wich showing image in popuped row.
